src/plugins/intel_gpu/src/graph/graph_optimizer/update_inner_program_io_map.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
nit: 2023

Reply:
I remained 2018 because it's just renamed from updated_loop_primitive_map. Should I remove 2018?